Gemfile.lock

```diff
@@ -1254,7 +1265,7 @@ DEPENDENCIES
   rspec-retry
   rspec-sidekiq
   rspec_junit_formatter
-  rswag-specs
+  rswag-specs!
```

Comment:
There's a linting error. Is it because of this line? Here's the error:
```
 > bundle install
/opt/hostedtoolcache/Ruby/3.3.3/x64/bin/bundle config --local path /home/runner/work/vets-api/vets-api/vendor/bundle
/opt/hostedtoolcache/Ruby/3.3.3/x64/bin/bundle config --local deployment true
Cache key: setup-ruby-bundler-cache-v6-ubuntu-22.04-x64-ruby-3.3.3-wd-/home/runner/work/vets-api/vets-api-with--without--only--Gemfile.lock-c9ad2785870190f70805105221380591329b5bba52f0e3f536054a375884bfcf
Received 0 of 194032542 (0.0%), 0.0 MBs/sec
Received 92274688 of 194032542 (47.6%), 44.0 MBs/sec
Cache Size: ~185 MB (194032542 B)
/usr/bin/tar -xf /home/runner/work/_temp/47c590d2-ec4a-49e0-949c-559c0514bb78/cache.tzst -P -C /home/runner/work/vets-api/vets-api --use-compress-program unzstd
Received 194032542 of 194032542 (100.0%), 61.7 MBs/sec
Cache restored successfully
Found cache for key: setup-ruby-bundler-cache-v6-ubuntu-22.04-x64-ruby-3.3.3-wd-/home/runner/work/vets-api/vets-api-with--without--only--Gemfile.lock-8523844df877ba4654c21875ffe959ac1b2147f0c65a9b095e3b0c09a3f244fa
/opt/hostedtoolcache/Ruby/3.3.3/x64/bin/bundle install --jobs 4
Your lockfile does not satisfy dependencies of "rswag-specs", but the lockfile
can't be updated because frozen mode is set
```

Reply:
The exclamation point is just for fetching via git instead of http, from what I can tell.